src/components/create-guild/Pagination.tsx

```diff
@@ -60,14 +60,14 @@ const Pagination = ({
           }
         >
           {prevStepText}
-        </Button>
+        </Button>*/}
         {!nextButtonHidden && (
           <Button
-            colorScheme="indigo"
+            colorScheme="green"
             isDisabled={nextButtonDisabled}
             onClick={nextStepHandler ?? nextStep}
           >
-            {nextStepText}
+            Add{/*nextStepText*/}
```

Comment:
Actually we don't need the Pagination component anymore, we can just use a simple Button instead as we discussed